docs/builder/manifest.md

```diff
@@ -0,0 +1,291 @@
+"background_color": "green"
+```
+
+!> PWABuilder expects a HEX color value for this field. You can convert an RGB color value to HEX [here.](https://www.rgbtohex.net/)
+
+### theme_color: `string`
+
+`theme_color` is an optional field that changes the default color used by certain OS features. For example, this would change the color of your title bar when the application is installed on Windows.
+
+```json
+"theme_color": "purple"
+```
+
+!> PWABuilder expects a HEX color value for this field. You can convert an RGB color value to HEX [here.](https://www.rgbtohex.net/)
+
+## Settings
+
+### start_url: `string`
+
+`start_url` is a required field that specifies that URL that will be launched when a user opens your application. This URL can either be an absolute or relative path.
+
+```json
+"start_url": "https://docs.pwabuilder.com"
+```
+
+### dir: `string`
+
+`dir` is an optional field that specifies the text direction for your PWA. 
+
+It has three values to choose from:
+
+* `auto` - No set directionality for your app.
+* `ltr` - Text will go from left to right.
+* `rtl` - Text will go from right to left.
+
+```json
+"dir": "ltr"
+```
+
+### scope: `string`
+
+`scope` is an optional field that defines which URL are within the navigation scope of your application. If the user navigates outside of your app's scope, the will be navigated to a normal browser window.
+
+`scope` can often just be set to the base URL of your PWA.
+
+```json
+"scope": "https://docs.pwabuilder.com"
+```
+
+### lang: `string`
+`lang` is an optional field that specifies the primary language of your app. The `Language` field expects a proper subtag for each langauge, and a list can be found [here.](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry)
+
+```json
+"lang": "en"
+```
+
+### orientation: `string`
+
+`orientation` is an optional field that specifies the default display orientation for your application. It can be any of the following values:
+
+* `any`
+* `natural`
+* `portrait`
+* `landscape`
+* `potrait-primary`
+* `portrait-secondary`
+* `landscape-primary`
+* `landscape-secondary`
+
+```json
+"orientation": "portrait"
+```
+
+
+### display: `string`
+
+`display` is an optional field that specifies the display mode that the website should default to.
+
+`display` can take any of the following values:
+
+* `browser`: The applications opens in a standard browser window.
+* `minimal-ui`: The application will open in a minimal browser window that still includes certain UI features, such as navigation.
+* `standalone`: The application will open in its own window with no browser UI elements.
+* `fullscreen`: The application will make use of all available display space.
+
+`display` will default to `browser` if not specified.
+
+```json
+"display": "standalone"
+```
+
+### display_override: `Array`
+
+`display_override` is similar to the `display` field, but allows the developer to select a fallback order for different display modes.
```

Comment:
Since the rest of file uses 'you' in place for 'the developer'
```suggestion
`display_override` is similar to the `display` field, but allows you to select a fallback order for different display modes.
```